Web/Assignment3/modalBootstrap.html

```diff
@@ -0,0 +1,144 @@
+					type="radio" id="terrible" name="radioBtn2" class="text-left" value=".btn-lg"/>&nbsp;.btn-lg</label>
+				<label class=" col-lg-2 col-md-2 col-sm-4 col-xs-12 text-primary"><input
+					type="radio" id="watchable" name="radioBtn2" class="text-left" value=".btn-sm" />&nbsp;.btn-sm</label>
+				<label class="col-lg-2 col-md-2 col-sm-offset-1 col-sm-4 col-xs-12 text-primary"><input
+					type="radio" id="bestEver" name="radioBtn2" class="text-left" value=".btn-xs" />&nbsp;.btn-xs</label>
+				<label class="col-lg-2 col-md-2 col-sm-4 col-xs-12 text-primary"><input
+					type="radio" id="bestEver" name="radioBtn2" class="text-left" value=".btn-block" />&nbsp;.btn-block</label>
+			</div>
+			
+			<div class="form-group row">
+				<label class="ques">Ques3: </label> <label class="quesText">Which of the following bootstrap style makes a set of buttons appear vertically stacked rather than horizontally?</label>
+			</div>
+			<div class="form-group row">
+				<label class=" col-lg-1 col-md-1 col-sm-1 col-xs-1" id="answer">Answer</label>
+				<label class="col-lg-2 col-md-2 col-sm-4 col-xs-12 text-primary"><input
+					type="radio" id="terrible" name="radioBtn3" class="text-left" value=".btn-group"/>&nbsp;.btn-group</label>
+				<label class=" col-lg-2 col-md-2 col-sm-4 col-xs-12 text-primary"><input
+					type="radio" id="watchable" name="radioBtn3" class="text-left" value=".btn-toolbar"/>&nbsp;.btn-toolbar</label>
+				<label class="col-lg-2 col-md-2 col-sm-offset-1 col-sm-4 col-xs-12 text-primary"><input
+					type="radio" id="bestEver" name="radioBtn3" class="text-left" value=".btn-group-lg" />&nbsp;.btn-group-lg</label>
+				<label class="col-lg-2 col-md-2 col-sm-4 col-xs-12 text-primary"><input
+					type="radio" id="bestEver" name="radioBtn3" class="text-left" value=".btn-group-vertical" />&nbsp;.btn-group-vertical</label>
+			</div>
+			
+			<div class="form-group row">
+				<label class="ques">Ques4: </label> <label class="quesText">What is mm jfdf kjdk jj dkdh lsjf fkdj fd?</label>
+			</div>
+			<div class="form-group row">
+				<label class=" col-lg-1 col-md-1 col-sm-1 col-xs-1" id="answer">Answer</label>
+				<label class="col-lg-2 col-md-2 col-sm-4 col-xs-12 text-primary"><input
+					type="checkbox" id="terrible" name="checkBtn1" class="text-left" value="Terrible"/>&nbsp;Terrible</label>
+				<label class=" col-lg-2 col-md-2 col-sm-4 col-xs-12 text-primary"><input
+					type="checkbox" id="watchable" name="checkBtn1" class="text-left" value="Watchable"/>&nbsp;Watchable</label>
+				<label class="col-lg-2 col-md-2 col-sm-offset-1 col-sm-4 col-xs-12 text-primary"><input
+					type="checkbox" id="bestEver" name="checkBtn1" class="text-left" value="Bestever"/>&nbsp;Bestever</label>
+				<label class="col-lg-2 col-md-2 col-sm-4 col-xs-12 text-primary"><input
+					type="checkbox" id="bestEver" name="checkBtn1" class="text-left" value="Bestever"/>&nbsp;Bestever</label>
+			</div>
+			
```

Comment:
1 Don't use same id for multiple element because ID id unique for each element if you want to give same functionality or same CSS then use class not ID